mdx-components.tsx

```diff
@@ -52,6 +51,7 @@ export function useMDXComponents(components: MDXComponents): MDXComponents {
     InlineFilter,
     MigrationAlert,
     YoutubeEmbed,
-    Banner
+    Banner,
+    ...components
```

Comment:
I think this was added by mistake by me, can we please remove `...components`

Reply:
Actually I think you removed it mistakenly so I added it back in this PR.